packages/nc-gui/components/cell/GeoData.vue

```diff
@@ -0,0 +1,156 @@
+<script lang="ts" setup>
+import type { GeoLocationType } from 'nocodb-sdk'
+import { Modal as AModal, latLongToJoinedString, useVModel } from '#imports'
+import GeoDataIcon from '~icons/mdi/map-marker'
```

Comment:
FYI. Icons from mdi are auto imported. Hence, L4 is not necessary since it is used in `<template/>` only.